Integration tests broken

Running this:

```
cd packages/ignite-integration-tests && npm run integration && cd ../..
```

Results in:

```
➜  ignite-integration-tests (master) npm run integration

> ignite-integration-tests@0.0.1 integration /Users/jh/Code/ReactNative/ignite/packages/ignite-integration-tests
> ava -s


  8 passed
  1 failed

  ignite-new › newInvalid › requires a name
  /Users/jh/Code/ReactNative/ignite/packages/ignite-integration-tests/out/new/test/ignite-new/newInvalid.test.js:22

  Test failed via t.fail()

  Test.<anonymous> (test/ignite-new/newInvalid.test.js:22:7)
  step (test/ignite-new/newInvalid.test.js:3:191)


npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "integration"
npm ERR! node v7.6.0
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! ignite-integration-tests@0.0.1 integration: `ava -s`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ignite-integration-tests@0.0.1 integration script 'ava -s'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the ignite-integration-tests package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ava -s
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ignite-integration-tests
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls ignite-integration-tests
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jh/Code/ReactNative/ignite/packages/ignite-integration-tests/npm-debug.log
```

Doctor:

```
System
  platform           darwin
  arch               x64
  cpu                8 cores                                                               Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  directory          /Users/jh/Code/ReactNative/ignite/packages/ignite-integration-tests

JavaScript
  node               7.6.0        /usr/local/bin/node
  npm                4.1.2        /usr/local/bin/npm
  yarn               0.21.3       /usr/local/bin/yarn

React Native
  react-native-cli   2.0.1

Ignite
  ignite             2.0.0-beta.6   /usr/local/bin/ignite

Android
  java               1.8.0_121    /usr/bin/java
  android home       -            /Users/jh/Library/Android/sdk

iOS
  xcode              8.2.1
```

--- a/packages/ignite-cli/src/commands/new.js
+++ b/packages/ignite-cli/src/commands/new.js
@@ -22,7 +22,7 @@ async function command (context) {
   const projectName = parameters.second
 
   // check for kebabs
-  const isKebabCase = not(isEmpty(match(/.-/g, projectName)))
+  const isKebabCase = not(isEmpty(match(/.-/g, projectName || '')))
 
   // camelCase the project name for user example
   const projectNameCamel = upperFirst(camelCase(projectName))

--- a/packages/ignite-cli/src/commands/plugin.js
+++ b/packages/ignite-cli/src/commands/plugin.js
@@ -150,4 +150,3 @@ module.exports = async function (context) {
       break
   }
 }
-

--- a/packages/ignite-cli/src/extensions/ignite/copyBatch.js
+++ b/packages/ignite-cli/src/extensions/ignite/copyBatch.js
@@ -39,7 +39,7 @@ module.exports = (plugin, command, context) => {
       if (await shouldGenerate(job.target)) {
         const currentPluginPath = ignitePluginPath()
         await template.generate({
-          directory: directory || currentPluginPath && `${currentPluginPath}/templates`,
+          directory: directory || (currentPluginPath && `${currentPluginPath}/templates`),
           template: job.template,
           target: job.target,
           props

--- a/packages/ignite-cli/src/lib/detectInstall.js
+++ b/packages/ignite-cli/src/lib/detectInstall.js
@@ -18,7 +18,7 @@ function detectInstall (context) {
   const { filesystem, parameters, ignite } = context
 
   // grab the plugin overrides
-  const pluginOverrides = ignite && ignite.pluginOverrides || []
+  const pluginOverrides = (ignite && ignite.pluginOverrides) || []
 
   /**
    * Is this a valid ignite plugin?

--- a/packages/ignite-cli/tests/cli/_mockGluegunBuilder.js
+++ b/packages/ignite-cli/tests/cli/_mockGluegunBuilder.js
@@ -1,5 +1,4 @@
 class MockGluegunBuilder {
-
   constructor () {
     this.reset()
   }

--- a/packages/ignite-dev-screens/templates/ComponentExamplesScreen.js
+++ b/packages/ignite-dev-screens/templates/ComponentExamplesScreen.js
@@ -8,7 +8,6 @@ import styles from './Styles/ComponentExamplesScreenStyles'
 import ExamplesRegistry from '../../App/Services/ExamplesRegistry'
 
 class ComponentExamplesScreen extends React.Component {
-
   renderAndroidWarning () {
     if (Platform.OS === 'android') {
       return (

--- a/packages/ignite-dev-screens/templates/FaqScreen.js
+++ b/packages/ignite-dev-screens/templates/FaqScreen.js
@@ -5,7 +5,6 @@ import { Images } from './DevTheme'
 import styles from './Styles/FaqScreenStyles'
 
 class ComponentExamplesScreen extends React.Component {
-
   render () {
     return (
       <View style={[styles.container, styles.mainContainer]}>

--- a/packages/ignite-dev-screens/templates/PluginExamplesScreen.js
+++ b/packages/ignite-dev-screens/templates/PluginExamplesScreen.js
@@ -13,7 +13,6 @@ import ExamplesRegistry from '../../App/Services/ExamplesRegistry'
 import styles from './Styles/PluginExamplesScreenStyles'
 
 class PluginExamplesScreen extends React.Component {
-
   render () {
     return (
       <View style={styles.mainContainer}>

--- a/packages/ignite-integration-tests/test/ignite-cli/noArgs.test.js
+++ b/packages/ignite-integration-tests/test/ignite-cli/noArgs.test.js
@@ -11,4 +11,3 @@ test('with no arguments', async t => {
   t.is(result.stderr, '')
   t.true(contains('https://infinite.red/ignite', result.stdout))
 })
-

--- a/packages/ignite-integration-tests/test/ignite-cli/version.test.js
+++ b/packages/ignite-integration-tests/test/ignite-cli/version.test.js
@@ -29,4 +29,3 @@ test('ignite version', async t => {
   const result = await execa(IGNITE, ['version'])
   t.is(result.stdout, VERSION)
 })
-

--- a/packages/ignite-integration-tests/test/ignite-new/newInvalid.test.js
+++ b/packages/ignite-integration-tests/test/ignite-new/newInvalid.test.js
@@ -25,3 +25,28 @@ test('requires a name', async t => {
     t.is(err.code, 5)
   }
 })
+
+test(`doesn't allow kebab-case`, async t => {
+  try {
+    await execa(IGNITE, ['new', 'chicken-kebab'])
+    t.fail()
+  } catch (err) {
+    t.is(err.stdout, 'Please use camel case for your project name. Ex: ChickenKebab\n')
+    t.is(err.code, 5)
+  }
+})
+
+test(`doesn't let you overwrite a folder`, async t => {
+  try {
+    jetpack.dir('Chicken')
+    await execa(IGNITE, ['new', 'Chicken'])
+    t.fail()
+  } catch (err) {
+    t.is(err.stdout, 'Directory Chicken already exists.\n')
+    t.is(err.code, 8)
+  } finally {
+    jetpack.remove('Chicken')
+  }
+})
+
+

--- a/packages/ignite-integration-tests/test/ignite-new/newInvalid.test.js
+++ b/packages/ignite-integration-tests/test/ignite-new/newInvalid.test.js
@@ -48,5 +48,3 @@ test(`doesn't let you overwrite a folder`, async t => {
     jetpack.remove('Chicken')
   }
 })
-
-

